projects/aca-playwright-shared/src/page-objects/components/manageRules/manage-rules-dialog.component.ts

```diff
@@ -29,15 +29,39 @@ export class ManageRulesDialogComponent extends BaseComponent {
   private static rootElement = 'aca-edit-rule-dialog';
 
   public createRuleButton = this.getChild('[data-automation-id="edit-rule-dialog-submit"]');
+  public cancelRuleButton = this.getChild('mat-dialog-actions button').filter({ hasText: 'Cancel' });
```

Comment:
should be checking the test ID without material selectors

Reply:
I will check that but many locators are missing test-automation-id and adding them is not a simple process sadly